web-client/scripts/npm-publish.sh

```diff
@@ -0,0 +1,5 @@
+./scripts/build-bundler.sh &&
+./scripts/build-web.sh &&
+cd dist &&
+npm publish --tag next &&
```

Comment:
`npm publish` supports a folder argument. Can we use it instead of `cd dist` & `cd ..`